kasa/cli.py

```diff
@@ -543,6 +543,23 @@ def _schedule_list(dev, type):
         click.echo(f"No rules of type {type}")
 
 
+@schedule.command(name="delete_rule")
```

Comment:
```suggestion
@schedule.command(name="delete")
```
maybe? This is just the command to execute (e.g., `schedule delete <id>`) and it feels more natural this way.